src/panels/lovelace/strategies/original-states-strategy.ts

```diff
@@ -53,7 +48,13 @@ export class OriginalStatesStrategy {
         hass.loadBackendTranslation("title"),
       ]);
 
-    const energyPrefs = energyPromise ? await energyPromise : undefined;
```

Comment:
We can just catch this in a try…catch here. No need to only start the promise after all the other info has fetched.